Gems/ROS2/Code/Include/ROS2/Sensor/SensorConfigurationRequestBus.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Interface/Interface.h>
+#include <ROS2/Sensor/SensorConfiguration.h>
+
+namespace ROS2
+{
+    //! Interface allows to change sensor configuration.
+
+    class SensorConfigurationRequest : public AZ::EBusTraits
```

Comment:
I don't think that entire sensor configuration should exposed to change. We are not handling changes in topic or QoS, which may be misleading. This interface needs to be limited to fields that 1) actually are effective in change 2) are clear in the meaning of change for users.